src/main/resources/config.sk

```diff
@@ -265,6 +265,15 @@ databases:
 
 		type: disabled # change to line below to enable this database
 		# type: SQLite
+		
+		variable re-save threshold: 1000
+		# The number of variables that are required to change in order for a re-save to occur.
+		# Setting this value lower is not recommended unless the server stores very few Skript variables, in which case it may be wise to lower
+		# this threshold to ensure that variables are always saved (prevents data loss during a crash).
+		
+		file re-write frequency in ticks: 6000
+		# The number of ticks that should pass before the database file is re-written.
+		# Setting this value lower is not recommended as saving large database files may cause server lag.

```

Comment:
Comment should explicitly tell how low it is safe to set. Also, I don't think the description about lag is quite right. Saving large databases takes *time*, and bad things happen if a database save has not succeeded before next one starts.

Reply:
>  bad things happen if a database save has not succeeded before next one starts.

that should be prevented